edit/networking.k8s.io.ingress/Certificate.vue

```diff
@@ -121,6 +122,10 @@ export default {
 .cert {
   position: relative;
 
+  .secret-name {
+    height: 54px;
```

Comment:
Shouldn't this use the $input-height you just defined?

Reply:
It should've been. I'll update.